lalrpop-util/src/lib.rs

```diff
@@ -182,6 +182,10 @@ pub struct ErrorRecovery<L, T, E> {
 ///
 /// // define a public module
 /// lalrpop_mod!(pub parser);
+///
+/// // specify attribute for the generated module
+/// // to suppress clippy vec_box warnings
+/// lalrpop_mod!(#[allow(clippy::vec_box)] parser);
```

Comment:
The attribute support is more generic than just suppressing clippy warnings, so I don't love the "to suppress clippy" line. Maybe something more generic "to add onto the generated code".

You aren't just limited to one attribute iirc.